src/components/CondensedTextView/CondensedTextView.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+
+import { connectTheme } from '../../utils';
+import { helpers } from '../../utils/';
+
+class CondensedTextView extends PureComponent {
+  static get propTypes() {
+    return {
+      enableCopy: PropTypes.bool,
+      leadText: PropTypes.string,
+      bodyText: PropTypes.string,
+      prefix: PropTypes.string,
+      suffix: PropTypes.string,
+      theme: PropTypes.object
+    };
+  }
+
+  onCopy(text) {
+    const textField = document.createElement('textarea');
+    textField.innerText = text;
+    document.body.appendChild(textField);
+    textField.select();
+    document.execCommand('copy');
+    textField.remove();
+  }
+
+  render() {
+    const {
+      enableCopy = true,
+      bodyText,
+      prefix,
```

Comment:
I wonder if it makes sense to default these to something? `4` as in your demo screenshot should be good. This reduces the amount of props someone _has_ to pass to the component to get it to work out of the box.

Reply:
Agreed.